pkgs/development/libraries/openvino/default.nix

```diff
@@ -0,0 +1,125 @@
+{ lib
+, autoPatchelfHook
+, stdenv
+, fetchgit
+, fetchurl
+, cmake
+, git
+, protobuf
+, tbb
+, opencv
+, unzip
+, shellcheck
+, python
+, cython ? null
+, pybind11 ? null
```

Comment:
```suggestion
, cython
, pybind11
```

Reply:
Without that when building openvino without python bindings it won't build

Reply:
If we use with python.pkgs those are no longer required.